src/pages/patientView/PatientViewPage.tsx

```diff
@@ -1,16 +1,18 @@
 import * as React from 'react';
 import * as ReactDOM from 'react-dom';
 import { connect } from 'react-redux';
-
+import { Component } from 'react';
 import ClinicalInformationContainer from './clinicalInformation/ClinicalInformationContainer';
 import PatientHeaderUnconnected from './patientHeader/PatientHeader';
 import {IPatientHeaderProps} from './patientHeader/PatientHeader';
 import {RootState} from "../../redux/rootReducer";
+import MSKPageDecorator from 'shared/components/MSKPageDecorator/MSKPageDecorator';
 
 interface IPatientViewPageProps {
     store?: RootState;
 }
 
+@MSKPageDecorator
```

Comment:
This might be a little too fancy, but what this decorator does is disable the page components render function when we detect that we are in the legacy site context.  In this context, it's then at the discretion of the legacy code to invoke the renderers which we expose on the window object.   
What I like about this solution is that we still have page specific bundles which include all the components for a given page.   In dev context, we can render these as per nomal in the page component.  But in legacy project, that may not be appropriate because we want to render the components on various places in the page and at various times.  This solution allows this freedom.